### ISSUE TITLE
Cannot find module: 'graphql-ruby-client/subscriptions/ActionCableLink'

### ISSUE THREAD

**Issue by dorianamouroux** [rmosolgo/graphql-ruby#2753]
Hi!

After updating the library from `1.6.8` to `1.7.3`, I get the following error:

```
Cannot find module: 'graphql-ruby-client/subscriptions/ActionCableLink'
```

In my code the import is like so:

```js
import ActionCableLink from 'graphql-ruby-client/subscriptions/ActionCableLink'
```

After investing a bit, I noticed in the version 1.6.8; the `subscriptions` was directly in the root folder of `node_modules/graphql-ruby-client` but in the 1.7.3, it's inside `node_modules/graphql-ruby-client`. 

I could fix it by having my import like so:

```js
import ActionCableLink from 'graphql-ruby-client/dist/subscriptions/ActionCableLink'
```

But I believe the import should be without the `dist` to comply with the docs and avoid breaking changes

**Comment by rmosolgo** [comment]
Thanks! I broke it during #2745 apparently. Let me find a better way of packaging it!

**Comment by rmosolgo** [comment]
I addressed this by moving all exports to the top level. That way, downstream projects don't have to worry about graphql-ruby-client's folder structure. (And, I think now that it's written with typescript, it can still support tree-shaking.)

I've updated the docs accordingly, and it should work better on 1.7.4 like this: 

```js
import { ActionCableLink } from 'graphql-ruby-client'
```

Let me know if you run into any other trouble, thanks again for reporting this!
